nvflare/fuel/hci/server/authz.py

```diff
@@ -62,9 +62,6 @@ def pre_command(self, conn: Connection, args: List[str]):
             return True
 
         if return_code == PreAuthzReturnCode.ERROR:
-            conn.append_error(
-                "Authorization error", meta=make_meta(MetaStatusValue.NOT_AUTHORIZED, "Authorization error")
-            )
             return False
```

Comment:
need to keep the original change

Reply:
No, this one must be removed, otherwise you will get two errors in the console output.

Reply:
Got it, thanks for the explanation.